### PR TITLE
Use yarn. Lock into a specific version of analyzer. Fix the build.

### PR DESCRIPTION
**Pull request by rictic**

### CODE REVIEW

**Comment by samccone** [comment]
:dancer:

File: yarn.lock [hunk 1]
```diff
@@ -0,0 +1,895 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (justinfagnani) [inline yarn.lock]: Remove. Lock files should only be for apps. We want to know if we break because of a dependency.

File: .travis.yml [hunk 2]
```diff
@@ -2,5 +2,9 @@ language: node_js
 node_js:
   - "4"
   - "6"
+before_install:
+- npm install -g  yarn
+install:
+- yarn install
```
Comment (justinfagnani) [inline .travis.yml]: we should look into caching this in Travis

**Review by justinfagnani — APPROVED** [review]

File: .travis.yml (hunk 2 above)
Comment (justinfagnani) [inline .travis.yml]: https://docs.travis-ci.com/user/caching/

File: .travis.yml [hunk 3]
```diff
@@ -2,5 +2,9 @@ language: node_js
 node_js:
   - "4"
   - "6"
+before_install:
+- npm install -g  yarn
```
Comment (justinfagnani) [inline .travis.yml]: `yarnpkg`?

**Pull request merged**